src/components/pages/order/OrderPage.js

```diff
@@ -1,17 +1,43 @@
-import React from "react";
+import { useState } from "react";
 import styled from "styled-components";
 import Navbar from "../order/Navbar/Navbar";
 import Main from "../order/Main/Main";
+import PanelContext from "../../context/OrderContext";
 import { theme } from "./../../../theme/index";
 
 export default function OrderPage() {
+  //state
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
+  const [isReduce, setIsReduce] = useState(false);
```

Comment:
```suggestion
  const [isHidden, setIsHidden] = useState(false);
```

"reduce" en programmation a une vrai signification. Ton nom porte à confusion. Préfère isHidden ou encore isCollapsed qui sont plus explicites.